pkg/monitortests/clusterversionoperator/legacycvomonitortests/operators.go

```diff
@@ -57,9 +187,26 @@ func testUpgradeOperatorStateTransitions(events monitorapi.Intervals, clientConf
 			return "We are not worried about Degraded=True blips for update tests yet.", nil
 		}
 
+		// we know the Status is not true and the Type is not degraded at this point indicating we are available=false
+		withinUpgradeWindowBuffer := isInUpgradeWindow(upgradeWindows, eventInterval) && eventInterval.To.Sub(eventInterval.From) < 10*time.Minute
+		if !withinUpgradeWindowBuffer {
+			switch operator {
+			// there are some known cases for authentication and image-registry that occur outside of upgrade window, so we will pass through and check for exceptions
+			case "authentication", "image-registry":
+				logrus.Infof("Operator %s is in Available=False state outside of upgrade window, but we will check for exceptions", operator)
+			default:
+				return "", nil
+			}
+		} else {
+			// SingleNode is expected to go Available=False for most / all operators during upgrade
+			if isSingleNode {
+				return fmt.Sprintf("Operator %s is in Available=False state running in single replica control plane, expected availability transition during upgrade", operator), nil
```

Comment:
@eggfoobar could you check my work and verify this is doing what you want.  'For all operators within an upgrade window on sno we allow Available=False'

Reply:
Looking good! I see the correct messages in the flakes, and most of those transitions are gone, running an upgrade test again to see if we can squeak a pass out of it but otherwise it's doing what we want. Really appreciate the help guys 😄